scripts/ensure_snap_builds.py

```diff
@@ -101,7 +101,7 @@ def ensure_lp_recipe(
         LOG.info(" Creating LP recipe %s", recipe_name)
         params = dict(**manifest)
         params.pop("auto_build_channels")
-        recipe = (not dry_run) and lp.snaps.new(project=lp_project, **params)
+        recipe = (not dry_run) and lp.client().snaps.new(project=lp_project, **params)
```

Comment:
i'd guess this is closer to correct right?

```suggestion
        recipe = (not dry_run) and client.snaps.new(project=lp_project, **params)
```